NuguAgents/Sources/CapabilityAgents/Nudge/NudgeAgent.swift

```diff
@@ -28,16 +28,22 @@ public class NudgeAgent: NudgeAgentProtocol {
     private let contextManager: ContextManageable
     private var nudgeInfo: [String: AnyHashable]?
     private var dialogRequestId: String?
+    private var currentPlayStacks: [(property: PlaySyncProperty, info: PlaySyncInfo)]?
     
     public let capabilityAgentProperty = CapabilityAgentProperty(category: .nudge, version: "1.0")
     public lazy var contextInfoProvider: ContextInfoProviderType = { [weak self] (completion) in
         guard let self = self else { return }
         
-        let payload: [String: AnyHashable?] = [
+        var payload: [String: AnyHashable?] = [
             "version": self.capabilityAgentProperty.version,
-            "nudgeInfo": self.nudgeInfo
         ]
         
+        if let nudgeInfo = self.nudgeInfo,
+           let currentPlayStacks = self.currentPlayStacks,
+           (currentPlayStacks.contains { $0.info.dialogRequestId == self.dialogRequestId }) {
+            payload["nudgeInfo"] = self.nudgeInfo
+        }
```

Comment:
nit. 동작상에는 아무 차이가 없지만 optional chaining 한 값을 사용해도 좋을 것 같습니다.
```suggestion
        if let nudgeInfo = self.nudgeInfo,
           let currentPlayStacks = self.currentPlayStacks,
           (currentPlayStacks.contains { $0.info.dialogRequestId == self.dialogRequestId }) {
            payload["nudgeInfo"] = nudgeInfo
        }
```

Reply:
감사합니다~